docs/docs/forms.mdx

```diff
@@ -2,13 +2,12 @@
 id: forms
 sidebar_label: Forms
 title: Forms
-description: Follow a rule-based process of information gathering using FormActions in open source bot framework Rasa.
+description: Follow a rule-based process of information gathering using forms in open source bot framework Rasa.
+abstract: One of the most common conversation patterns is to collect a few pieces of
+          information from a user in order to do something (book a restaurant, call an
+          API, search a database, etc.). This is also called **slot filling**.
```

Comment:
@melindaloubser1 i don't think this is gonna work, it has to be on one line otherwise livedocs throws an error. might be worth checking with `make livedocs` locally to see that the formatting also renders correctly